contributor-ladder.md

```diff
@@ -0,0 +1,157 @@
+
+* Privileges:
+    * May be assigned Issues and Reviews
+    * May give commands to CI/CD automation
+    * Entitled to vote in the [TODO: appropriate election]
+    * Can be added to [TODO: Repo Host] teams
+    * [TODO: Other Privileges]
+
+
+The process for a Contributor to become an Organization Member is as follows:
+
+<!-- the process of becoming an organization member is going to depend strongly on how your project manages its infrastructure. TO DO: Project leads to fil in exact details of how a contributor becomes an organization member-->
+* High skill for development and meaninful contribution to a project(s)
+* Demonstrates leadership capabilities 
+* Willing to mentor other contributors
+
+### Reviewer
+<!--TODO: project leads to fill in exact details of this role for your project-->
+Description: A reviewer approves pull requests before they're merged by maintainers. May only have responsibility over a subarea of the project initially, where they have demonstrated domain expertise. A reviewer must meet the responsibilites and has the requirements of a Contributor, plus:
+
+* Responsibilities include:
+    * Reviewing pull requests in a timely manner
+    * Following the reviewing guide
+* Requirements:
+    * Experience as a Contributor
+* Additional privileges:
+    * Has Github rights to approve pull requests
+
+### Maintainer
+<!--TODO: project leads to fill in exact details of this role for your project-->
+Description: A Maintainer is a contributor with commit access. Decisions and prioritizes about the project. Maintainer roles may include Community Managers, Project Managers, Release Managers, Docs Managers, and Subproject Lead. An Maintainer must meet the responsibilites and has the requirements of an Reviewer, plus:<!-- sometimes also a project lead - if so, please state in the role defintion -->
+
+* Criteria
+    * Willing to accept additional responsibilites
+    * Ability to keep commitments and follow-up
+    * Demonstrates project leadership
+
+#### Code Maintainer -> expansion pack
+* Criteria
+    * Experience as a contributor
+    * Experience as an reviewer
+* Responsibilities include:
+    * Must follow the [CNCF CoC](https://github.com/cncf/foundation/blob/master/code-of-conduct.md)
+    * Merge pull requests in a timely manner
+    * Following the reviewing guide
+    * Helping new contributors
+    * Executing conference/speaking obligations
+    * Handling konveyor relations
+    * Speaking on behalf of the project publically
+* Additional privileges:
+    * Commit access to project repo(s)
+
+## Inactivity
+<!--TODO: project leads to fill in exact details for how you measure inactivity for your project-->
+It is important for contributors to be and stay active to set an example and show committment to the project. Inactivity is harmful to the project as it may lead to unexpected delays, contributor attrition, and a lost of trust in the project. 
```

Comment:
```suggestion
It is important for contributors to be and stay active to set an example and show commitment to the project. Inactivity is harmful to the project as it may lead to unexpected delays, contributor attrition, and loss of trust in the project. 
```